README.md

```diff
@@ -1,8 +1,42 @@
-Foreman Remote Execution Commands for Hammer CLI
-================================================
+CLI Plugin for Foreman Remote Execution
+=======================================
 
 This [Hammer CLI](https://github.com/theforeman/hammer-cli) plugin contains commands for [foreman_remote_execution](https://github.com/theforeman/foreman_remote_execution).
 
+Examples
+--------
+
+### Create a Template
+
+```
+hammer job-template create --file /tmp/template.txt --name "Ping a Host"\
+  --provider-type Ssh --job-name "Ping"
+```
+
+### Create a Template Input
+
+```
+hammer job-template input create --job-template "Ping a Host" --name hostname\
+  --input-type user --options www.google.com,www.facebook.com,localhost
+```
+
+### Run a Job
+
+```
+hammer job-invocation create --job-name "Ping" --job-template "Ping a Host"\
+  --inputs hostname=www.google.com --search-query "name ~ rex01"\
```

Comment:
And also how to properly escape quotation marks would come in handy since naive `--input "command=echo 'this is a test'"` somehow makes the second `'` disappear